documentation/src/docs/recovery-procedures/metadata-table-segment-recovery.md

```diff
@@ -0,0 +1,120 @@
+<!--
+Copyright Pravega Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+# Metadata Table Segment Attribute Index Recovery
+
+
+The below outlined steps refer to recovery of Attribute Index Table Segments but only in cases of Metadata Segments which are a special case, as they need to
+be handled a little differently. Metadata Segments in Pravega are Table Segments, that hold metadata about all other "non-metadata" Segments, which
+include the internal Segments as well as the User Segments. Metadata about these Metadata Segments themselves in Pravega, are stored in Pravega Journals. More details
+about the why and how part of Journals can be found in this PDP [here](https://github.com/pravega/pravega/wiki/PDP-34-(Simplified-Tier-2)#why-slts-needs-system-journal).
+Pravega Journals, is concretely what we would be attempting to update as part of the recovery procedure in this document.
+
+
+Please note that the steps outlined below are in continuation to the ones outlined as part of the recovery procedure described [here](https://github.com/pravega/pravega/blob/master/documentation/src/docs/recovery-procedures/table-segment-recovery.md).
+That is, we need to perform some of the steps mentioned in the above referenced doc, before continuing any of the steps present here.
+
+
+It is precisely at this point [here](https://github.com/pravega/pravega/blob/master/documentation/src/docs/recovery-procedures/table-segment-recovery.md#important) in 
+the Table Segment Recovery procedure, that one would have to jump to the below steps in the case of dealing with Metadata Segments.
+
+
+# Detailed Steps
+1) Once we determine that the Table Segment under repair is a Metadata Table Segment, we copy over the Attribute Index Chunks generated 
+   in the output directory of [Step 4](https://github.com/pravega/pravega/blob/a5088a464275d5ea90adb09ac39027332e87a8e3/documentation/src/docs/recovery-procedures/table-segment-recovery.md?plain=1#L129) to a separate directory of our choice.
```

Comment:
This link is showing plain md code, not the rendered one.

Reply:
as i am linking to specific locations within the doc, the view just lands automatically in "source blob" mode( and not the "rendered blob" mode).